src/app/tags/tags/tags.component.ts

```diff
@@ -71,7 +74,15 @@ export class TagsComponent implements OnInit {
       });
```

Comment:
- `if (transactions.length > 0)` the same as `if (transactions.length)`
- you will try to delete transactions/tag on each transactions list change. try to use operator `first` for observable

Reply:
1. It makes code more readable
2. I do unsubscribe after delete operation completes

Reply:
2. Try to log your deletion. You will see much more actions then you expect